diskplorer.py

```diff
@@ -42,6 +42,10 @@
                     help='Directory to place fio job files (default: files will not be kept)')
 parser.add_argument('--result-file', type=str, required=True,
                     help='Results, in fio json+ format')
+parser.add_argument('--run-sequential-trim', action='store_true',
+                    help='Run sequential trim workload in addition to read/write ones to check its impact.')
```

Comment:
"run" is awkward and implied, you can remove it.

Reply:
Fixed